docs/usage/configuration-options.md

```diff
@@ -1556,6 +1556,18 @@ To adjust it down to 10s for all queries, do this:
 }
 ```
 
+### httpsCertificateAuthority
+
+`httpsCertificateAuthority` overrides any trusted CA certificate.
```

Comment:
Could you provide more clarity about what this means? e.g. does it mean that it overrides the `httpsCertificate` field?

Reply:
sure

Reply:
My understanding is that node ships its own list of root CAs ([here](https://github.com/nodejs/node/blob/main/src/node_root_certs.h)), which `got` reuses. Providing a custom trust store here should only be needed in exceptional cases, if a custom, generally untrusted CA is used for mTLS.